inspectit-oce-documentation/src/docs/asciidoc/_includes/configuration/configuration-basics.adoc

```diff
@@ -0,0 +1,69 @@
+=== Configuration Basics
+
+inspectIT OCE tries to implement the zero-configuration approach but lets you externalize the configuration and influence every bit of its configuration if this is required.
+Internally it uses the Spring-based `PropertySource` order to allow overriding of configuration values.
```

Comment:
Is this implementation detail really required here?

Reply:
I wanted to mention this because I think a lot of people are used to ways of defining properties in Spring-based application and that this could be helpful?